xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml

```diff
@@ -191,113 +191,115 @@
         &lt;input type="hidden" name="userScope" value="$!escapetool.xml($request.userScope)" /&gt;
         #addMembersAsHiddenInput()
-        ### WIZARD FIRST HALF ###
-        &lt;div class="half"&gt;
-          ### PRETTY NAME ###
-          &lt;div&gt;
-            &lt;dl&gt;
-              &lt;dt&gt;
-                &lt;label for="prettyname"&gt;$services.localization.render('platform.wiki.prop.wikiprettyname')#required()&lt;/label&gt;
-                &lt;span class="xHint"&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikiprettyname')&lt;/span&gt;
-                &lt;span id="wikiprettynamevalidation" class="xErrorMsg"&gt;&lt;/span&gt;
-              &lt;/dt&gt;
-              &lt;dd&gt;
-                &lt;input size="60" id="prettyname" name="wikiprettyname" type="text" value="$!escapetool.xml($request.getParameter('wikiprettyname'))"/&gt;
-              &lt;/dd&gt;
-            &lt;/dl&gt;
-          &lt;/div&gt;
-          ### WIKI NAME ###
-          &lt;div&gt;
-            &lt;dl&gt;
-              &lt;dt&gt;
-                &lt;label for="wikiname"&gt;$services.localization.render('platform.wiki.prop.wikiname')#required()&lt;/label&gt;
-                &lt;span class='xHint'&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikiname')&lt;/span&gt;
-              &lt;/dt&gt;
-              &lt;dd&gt;
-                &lt;input size="40" id="wikiname" name="wikiname" type="text"  value="$!escapetool.xml($request.wikiname)"/&gt;
-                &lt;span id="wikinamevalidation" class="xErrorMsg"&gt;&lt;/span&gt;
-              &lt;/dd&gt;
-            &lt;/dl&gt;
-          &lt;/div&gt;
-          ### WIKI ALIAS ###
-          #if(!$services.wiki.isPathMode())
+        &lt;div class="row"&gt;
+          ### WIZARD FIRST HALF ###
+          &lt;div class = "col-xs-12 col-sm-6 col-md-6"&gt;
+            ### PRETTY NAME ###
             &lt;div&gt;
               &lt;dl&gt;
                 &lt;dt&gt;
-                  &lt;label for="wikialias"&gt;$services.localization.render('platform.wiki.prop.wikialias')#required()&lt;/label&gt;
-                  &lt;span class='xHint'&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikialias')&lt;/span&gt;
+                  &lt;label for="prettyname"&gt;$services.localization.render('platform.wiki.prop.wikiprettyname')#required()&lt;/label&gt;
+                  &lt;span class="xHint"&gt;$services.localization.render('platform.wiki.create.wizard.desc.wikiprettyname')&lt;/span&gt;
+                  &lt;span id="wikiprettynamevalidation" class="xErrorMsg"&gt;&lt;/span&gt;
                 &lt;/dt&gt;
                 &lt;dd&gt;
-                  &lt;input size="40" id="wikialias" name="wikialias" type="text"  value="$!escapetool.xml($request.wikialias)"/&gt;
+                  &lt;input size="60" id="prettyname" name="wikiprettyname" type="text" value="$!escapetool.xml($request.getParameter('wikiprettyname'))"/&gt;
```

Comment:
We could use the same size for input type text. In http://platform.xwiki.org/xwiki/bin/view/DevGuide/VerticalForms, the size is recommended to '60'